geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -1188,11 +1210,51 @@ public void endPutLocal(long start) {
     stats.incLong(putLocalTimeId, delta);
   }
 
+  public long startBucketClear() {
+    incBucketClearsInProgress();
+    return clock.getTime();
+  }
+
+  public void endBucketClear(long start) {
+    long delta = clock.getTime() - start;
```

Comment:
check "if (clock.isEnabled)" and only calc delta and inc bucketClearTimeId if it is

Reply:
will do.